### PR TITLE
tests: check that handlers are able to use 'include'

### PR DESCRIPTION
**Pull request by pilou-**
Following https://github.com/ansible/ansible/pull/11520#issuecomment-124164941

### CODE REVIEW

**Comment by pilou-** [comment]
Rebased.

File: test/integration/targets/handlers/test_handlers_include.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+- name: verify playbook handlers can includes tasks
```
Comment (mscherer) [inline test/integration/targets/handlers/test_handlers_include.yml]: I think you got the description mixed, could you fix it ?

File: test/integration/targets/handlers/test_handlers_include.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  hosts: testhost
+  tasks:
+    - debug: msg="main task"
+      changed_when: True
+      notify: test handler
+      tags: ['scenario1']
```
Comment (mscherer) [inline test/integration/targets/handlers/test_handlers_include.yml]: I would also prefer a more explicit name for the tag

File: test/integration/targets/handlers/test_handlers_include.yml (hunk 1 above)
Comment (pilou-) [reply test/integration/targets/handlers/test_handlers_include.yml]: Done

File: test/integration/targets/handlers/test_handlers_include.yml (hunk 2 above)
Comment (pilou-) [reply test/integration/targets/handlers/test_handlers_include.yml]: Done

**Pull request merged**